nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -34,49 +40,45 @@ BtNavigator::execute(const nav2_tasks::NavigateToPoseCommand::SharedPtr command)
     command->pose.position.x, command->pose.position.y);
 
   // Get the current pose from the robot
-  geometry_msgs::msg::PoseWithCovarianceStamped::SharedPtr current;
+  auto current = std::make_shared<geometry_msgs::msg::PoseWithCovarianceStamped>();
```

Comment:
In simple navigator, this is called `current_pose` which I kind of prefer to just `current`